### ISSUE TITLE
Ascending progression

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9922]
**Preferred term label:**
Ascending progression

**Synonyms**
Progression of symptoms caudal to cranial

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lambert-Eaton myasthenic syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000305
- label: Proximal to distal progression
- parents: [Temporal pattern (HP:0011008)](https://hpo.jax.org/app/browse/term/HP:0011008)
- pmid: 
- def: Refers to symptomatology that begins in the proximal limbs and subsequently affects the distal limbs.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9922
- synonyms:
